mobile/src/main/java/org/openhab/habdroid/core/NotificationHelper.kt

```diff
@@ -140,17 +147,38 @@ class NotificationHelper(private val context: Context) {
             .setContentIntent(contentIntent)
             .build()
 
-        return makeNotificationBuilder(channelId, message.createdTimestamp)
+        val builder = makeNotificationBuilder(channelId, message.createdTimestamp)
             .setLargeIcon(iconBitmap)
-            .setStyle(NotificationCompat.BigTextStyle().bigText(message.message))
             .setSound(context.getPrefs().getNotificationTone())
+            .setContentTitle(message.title)
             .setContentText(message.message)
-            .setSubText(message.severity)
+            .setSubText(message.tag)
             .setContentIntent(contentIntent)
             .setDeleteIntent(deleteIntent)
             .setVisibility(NotificationCompat.VISIBILITY_PRIVATE)
             .setPublicVersion(publicVersion)
-            .build()
+
+        val messageImage = if (message.mediaAttachmentUrl != null) {
+            ConnectionFactory.waitForInitialization()
+            ConnectionFactory.primaryUsableConnection?.connection?.let {
+                message.loadImage(it, context, 99)
```

Comment:
@maniac103 I'm not sure about what size should be used here. Maybe the screen width?